package.json

```diff
@@ -26,8 +26,10 @@
   },
   "homepage": "https://github.com/josh/selector-observer#readme",
   "files": [
+    "dist/index.esm.d.ts",
```

Comment:
@joshaber should I also add a `"types": "./dist/index.esm.d.ts"` to the `package.json`?

Reply:
Nice work 🤘 It seems to work just fine without it.